Kernel/Prekernel/Arch/aarch64/boot.S

```diff
@@ -15,6 +15,12 @@ start:
   and x13, x13, 0xff
   cbnz x13, _ZN9Prekernel4haltEv
 
+  // Enable the use of va_list in EL1
+  // by disabling the trap of accessing FP and SIMD registers.
+  mrs x0, CPACR_EL1
+  orr x0, x0, #(0b11 << 20)
```

Comment:
Why not do this in the Prekernel `init()`? We try to keep the asm stub as small as possible. (See: `Kernel/Prekernel/Arch/aarch64/init.cpp`)